meta-fobnail-distro/recipes-fobnail/libcoap/libcoap_%.bb

```diff
@@ -0,0 +1,37 @@
+LICENSE = "Unknown"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=e44b3af4925ec58e9f49b9ff143b5493 \
+                    file://COPYING;md5=faed8f005d476edd3f250599a4bb9a75 \
+                    file://ext/tinydtls/LICENSE;md5=ffb073dbb36e7ec5e091047332f302c5"
+
+SRC_URI = "gitsm://github.com/obgm/libcoap.git;protocol=https;branch=release-4.3.0"
+
+PV = "4.3.0+git${SRCPV}"
```

Comment:
@kotylamichal after the recipe name will be changed this will not be necessary, the PV will be 4.3.0

Reply:
done